src/adapters/humctl/HumctlAdapter.ts

```diff
@@ -61,6 +61,7 @@ export class HumctlAdapter implements IHumctlAdapter {
       result = await exec(humctlFilePath, command, {
         env: await this.prepareEnvVars(),
       });
+      /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Shouldn't we fix them 😅

Reply:
Hmm... some of them for sure. Some of them are `any` for a reason. The lint-staged that you proposed checking only files that are on staging so we can gradually fix all the warning, adding new features